module-infrastructure/security/src/main/kotlin/site/yourevents/SecurityConfig.kt

```diff
@@ -14,11 +16,17 @@ import org.springframework.web.cors.reactive.UrlBasedCorsConfigurationSource
 @Configuration
 @EnableWebSecurity
 class SecurityConfig(
+    @Value("\${app.server.url}")
+    private val serverUrl: String
 ) {
+
     @Bean
```

Comment:
기존 컨벤션에 맞추어, 개행은 제거해주는 것이 좋을 것 같습니다!
```suggestion
    @Value("\${app.server.url}")
    private val serverUrl: String
) {
    @Bean
```

Reply:
수정하였습니다!